vector/src/main/java/im/vector/app/features/settings/VectorSettingsPreferencesFragment.kt

```diff
@@ -99,6 +102,10 @@ class VectorSettingsPreferencesFragment :
             }
         }
 
+        findPreference<VectorPreferenceCategory>(VectorPreferences.SETTINGS_PREF_SPACE_SHOW_ALL_ROOM_IN_HOME_CATEGORY)!!.let { pref ->
```

Comment:
Maybe casting to `Preference` is enough. But not a blocker.
```suggestion
        findPreference<Preference>(VectorPreferences.SETTINGS_PREF_SPACE_SHOW_ALL_ROOM_IN_HOME_CATEGORY)!!.let { pref ->
```